src/components/parcourslist/ParcoursList.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import * as _ from 'lodash';
+import { makeStyles } from '@material-ui/core/styles';
+import Parcours from '../parcours/Parcours';
+import Leaflet from '../map/Map';
+import { ApiContext } from '../../contexts/ApiContext';
+
+const CoursesListStyles = makeStyles(() => ({
+  parcours: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+  },
+  parcoursMap: {
+    height: '350px',
+  },
+  parcoursInfo: {
+    display: 'flex',
+    height: '80%',
+    padding: '20px 20px 100px 20px',
+  },
+  parcoursCapsule: {
+    display: 'flex',
+    overflow: 'auto',
+    padding: '0 17px 17px 0',
+  },
+  '@global': {
+    '*::-webkit-scrollbar': {
+      display: 'none',
+    },
+  },
+}));
+
+const ParcoursList = () => {
+  const classes = CoursesListStyles();
+  const { course } = useContext(ApiContext);
+  const parcours = [...course];
+
+  const uniqueCourseByID = _.uniqBy(parcours, 'course_id');
+  console.log(uniqueCourseByID);
+  console.log(parcours);
+
+  return (
+    <div className={classes.parcours}>
+      <div className={classes.parcoursMap}>
+        <Leaflet />
+      </div>
+      <div className={classes.parcoursInfo}>
+        {parcours && (
+          <div className={classes.parcoursCapsule}>
+            {uniqueCourseByID.map((parc, index) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <Parcours key={index} parcours={parc} fullparcours={parcours} />
+            ))}
+          </div>
```

Comment:
is there an other fix for line 52 ?

Reply:
- Your Parcours probably have a `id` field in DB, which is usable as a `key` value
- Don't use `eslint-disable` without discussing it beforehand with your SM and/or instructor
- A quick search explained the problem with [this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md) (and honestly, it's an error I was unaware of until now!!):

Reply:
Fixed. I replace index by parc.id and it works.